srcpkgs/python3-tasklib/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'python3-tasklib'
+pkgname=python3-tasklib
+version=2.4.3
+revision=1
+wrksrc="${pkgname#*-}-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3 task python3-tzlocal python3-pytz"
+checkdepends="task"
+short_desc="Library for interacting with taskwarrior databases (Python3)"
+maintainer="Cullen Ross <cullenrss@gmail.com>"
+license="BSD-3-Clause"
+homepage="https://github.com/robgolding/tasklib"
```

Comment:
This seems to redirect to https://github.com/robgolding/tasklib for me.

Reply:
Changed the link the the Gothenburg one. Great catch!